### ISSUE TITLE
Only show the overflowWidget when there is an actual overflow

### ISSUE THREAD

**Issue by nietsmmar** [elileo1/extended_wrap#4]
At the moment the `overflowWidget` is always shown. Also if there is not even an overflow, or even if my wrap is empty. How can I make it only show, when there is an actual overflow?

**Comment by elileo1** [comment]
ok , i will update it

**Comment by elileo1** [comment]
the example can do it , you can see it , delete some data and you will see

**Comment by nietsmmar** [comment]
What did you change? Because for me it is not working. It is always showing my `overflowWidget`

**Comment by elileo1** [comment]
minLines attr
first time pack up state must equals maxLines

**Comment by elileo1** [comment]
you can find it in _MyHomePageState, initState funcation

**Comment by elileo1** [comment]
you can find it in _MyHomePageState initState function

**Comment by nietsmmar** [comment]
For me it is not working. I loaded the `example`. But let's say I want to set `minLines` to **3**. And I comment out the last three items from the `resultList` in the `example`. I get this: 

![Screenshot_1671550146](https://user-images.githubusercontent.com/9265181/208704194-6855b522-7d24-4af3-8edd-b43de7d14a78.png)

The `expandButton` is shown, altough there is nothing to expand. And if I click it, it does not show anything new, just changing the arrow in the `expandButton`.

**Comment by elileo1** [comment]
show me the initState function code

**Comment by elileo1** [comment]
ok , i will fix it

**Comment by elileo1** [comment]
I've fixed it ， version 0.1.3

**Comment by nietsmmar** [comment]
Sadly it is still not working for me. This specific example is fixed now, but there are still usecases where it does not work. For Example (on my emulator) using this:

```
List<String> resultList = [
    "穿搭冬季",
    "2021秋冬流行穿搭",
    "穿搭",
    "家常菜",
    "文案",
    "头像",
    "健身",
    "学习",
    "跟井柏然学花衬衫叠穿",
    "学习",
    "学习",
  ];
```

resulting in this:

![Screenshot_1671641557](https://user-images.githubusercontent.com/9265181/208960748-b7701325-8417-47f7-bc74-eb0cd3d238ff.png)

**Comment by elileo1** [comment]
ok , i will review the code

**Comment by elileo1** [comment]
I've fixed it ， version 0.1.4
Thank you for your feedback， you can try it again

**Comment by nietsmmar** [comment]
Great :) It seems to work now. Thank you!
